tests/Api.IntegrationTests/AuthorizationTests.cs

```diff
@@ -59,9 +62,118 @@ public async Task ValidateThatEndpointsHaveProtection()
         }
     }
 
+    [Fact]
+    public async Task ValidateThatMissingApiSecretThrows()
+    {
+        var request = new HttpRequestMessage(HttpMethod.Get, "/credentials/list?userId=1");
+        request.Headers.Add("Accept", "application/json");

```

Comment:
Should we add the `Accept` header ad a DefaultHeader for the entire client in the constructor, or alternatively even override `CreateClient` and do it in `TestWebApplicationFactory`?

Reply:
Yeah, I think we can add it to `TestWebApplicationFactory `.

But I'm also a little bit worried -- I think we should default the entire API to have accept:application/json unless the caller sends us anything different.